src/main/java/org/apache/commons/validator/routines/checkdigit/ModulusCheckDigit.java

```diff
@@ -185,6 +185,16 @@ protected int toInt(final char character, final int leftPos, final int rightPos)
         throw new CheckDigitException("Invalid Character[" + leftPos + "] = '" + character + "'");
     }
 
+    /**
+     * A convenient method to check empty Strings.
+     *
+     * @param code The code to check
+     * @return {@code true} if code is null or empty, otherwise {@code false}
+     */
+    protected boolean isEmpty(final String code) {
+        return code == null || code.isEmpty();
```

Comment:
Oops, I realized yesterday we already have this: `org.apache.commons.validator.GenericValidator.isBlankOrNull(String)` so replace call sites to this with calls to `isBlankOrNull()`. Sorry about that!